edumanage/models.py

```diff
@@ -520,6 +716,12 @@ class InstitutionDetails(models.Model):
         blank=True,
         help_text=_('The primary, registered domain name for your institution, eg. example.com.<br>This is used to derive the Operator-Name attribute according to RFC5580, par.4.1, using the REALM namespace.')
     )
+    venue_info = models.CharField(
+        max_length=7,
+        blank=True,
+        validators=[validate_venue_info],
+        db_column='inst_type'
+    )
```

Comment:
Why is the field called`venue_info` but stored in db_column `inst_type` ?  That just looks like unnecessary complexity - can't we just use one name for both?

I expect there is a reasonable answer to this - just asking because it struck me as odd.

Reply:
The eduroam db spec, both the abstract version and the XML (JSON) schema, defines some quite generic field names, which felt too confusing (or even technically inappropriate) to use for our Django model fields. We have always followed the convention of using the actual name in the spec for `db_column`, if at all possible, as this is hidden from casual usage, but we still get to record the mapping in there. And now we will actually get to use this mapping, as you will see in upcoming commits. I do understand however that as new eduroam db versions arise, such a mapping mechanism is inadequate and should be replaced by something that also considers the EDB version. I could not tackle this at this stage, however.

Reply:
Sure - also happy with this - glad to hear you did consider this carefully.